src/metrics/extract_ttfs.py

```diff
@@ -78,6 +72,16 @@ def main():
                 if (t.platformID == 0)
                 or (t.platformID == 3 and t.platEncID in (1, 10))]
 
+        if font in metrics_to_extract:
+            chars = metrics_to_extract[font]
+            chars[u"\u0020"] = None  # space
+            chars[u"\u00a0"] = None  # nbsp
+        else:
+            chars = {
+                u"\u0020": None,  # space
+                u"\u00a0": None,  # nbsp
+            }
+
```

Comment:
Could we remove this repetition? In the `else:` case we could just set `chars = {}` and then move the space settings out of the `if:` case.

Reply:
Actually, instead of the `if`, we could use `chars = metrics_to_extract.get(font, {})`